pom.xml

```diff
@@ -132,6 +132,16 @@
                 <artifactId>siddhi-execution-list</artifactId>
                 <version>${siddhi.execution.list}</version>
             </dependency>
+            <dependency>
+                <groupId>commons-io</groupId>
+                <artifactId>commons-io</artifactId>
+                <version>2.6</version>
```

Comment:
Don't put version directly here, replace the version with a variable and define the version under the property tag

Reply:
fixed in https://github.com/siddhi-io/siddhi-io-file/pull/82/commits/216bbff54ceeaef2da8cba05aab1111e245f6e91

Reply:
> fixed in https://github.com/siddhi-io/siddhi-io-file/pull/82/commits/95d54cbe3dfd439c8870b01db7756a978bf58d19